src/components/MovieInfo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+
+class MovieInfo extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      movie: this.props.movie,
+    }
+  }
+
+  render() {
```

Comment:
Can you please use https://material-ui.com/getting-started/installation/ instead of the raw html elements?